contributors/devel/cherry-picks.md

```diff
@@ -1,44 +1,33 @@
-1. `release-note` labeled PRs generate a release note using the PR title by
-   default OR the release-note block in the PR template if filled in.
-  * See the [PR template](https://git.k8s.io/kubernetes/.github/PULL_REQUEST_TEMPLATE.md) for more details.
-  * PR titles and body comments are mutable and can be modified at any time
-    prior to the release to reflect a release note friendly message.
-
-### How do cherrypick-candidates make it to the release branch?
-
-1. **BATCHING:** After a branch is first created and before the X.Y.0 release
-  * Branch owners review the list of `cherrypick-candidate` labeled PRs.
-  * PRs batched up and merged to the release branch get a `cherrypick-approved`
-label and lose the `cherrypick-candidate` label.
-  * PRs that won't be merged to the release branch, lose the
-`cherrypick-candidate` label.
-
-1. **INDIVIDUAL CHERRYPICKS:** After the first X.Y.0 on a branch
-  * Run the cherry pick script. This example applies a master branch PR #98765
-to the remote branch `upstream/release-3.14`:
-`hack/cherry_pick_pull.sh upstream/release-3.14 98765`
-  * Your cherrypick PR (targeted to the branch) will immediately get the
-`do-not-merge/cherry-pick-not-approved` label. The branch owner will triage PRs
-targeted to the branch and label the ones to be merged by applying the `lgtm`
-label.
-
-There is an [issue](https://github.com/kubernetes/kubernetes/issues/23347) open
-tracking the tool to automate the batching procedure.
+This document explains how cherry picks are managed on release
+branches within the Kubernetes projects.
+
+## Prerequisites
+ * [Contributor License Agreement](http://git.k8s.io/community/CLA.md)
+   is considered implicit for all code within cherry-pick pull requests,
+   ***unless there is a large conflict***.
+ * A pull request merged against the master branch.
+ * [Release branch](https://git.k8s.io/release/docs/branching.md) exists.
+ * The normal git and GitHub configured shell environment for pushing
+   to your kubernetes `origin` fork on GitHub and making a pull request
+   against a configured remote `upstream` that tracks
+   "https://github.com/kubernetes/kubernetes.git", including
+   `GITHUB_USER`.
+ * Have `hub` installed, which is most easily installed via `go get
+   github.com/github/hub` assuming you have a standard golang development
+   environment.
+
+## Initiate a Cherry Pick
+ * Run the [cherry pick script](https://git.k8s.io/kubernetes/hack/cherry_pick_pull.sh).
+   This example applies a master branch PR #98765 to the remote branch
+   `upstream/release-3.14`: `hack/cherry_pick_pull.sh upstream/release-3.14
+   98765`
+ * Your cherrypick PR targeted to the release branch will immediately get the
+   `do-not-merge/cherry-pick-not-approved` label. The release branch owner
+   will triage PRs targeted to the branch.  Normal rules apply for code
+   reviewers applying the `/lgtm` label as appropriate, with the branch
+   owner applying the `/approve` label.
```

Comment:
Sorry, I should have been clearer.. the branch manager will apply "cherry-pick-approved", but OWNERS will still need to "/approve" the PR.

Reply:
So that makes sense, but...is that what's actually implemented?  I'm not seeing that in prow.

Reply:
The `cherrypick-approved` label doesn't have an associated prow command. It's added manually by the branch manager.

Example PR that shows the process: https://github.com/kubernetes/kubernetes/pull/62847

Reply:
Got it!  Amended.